bcrhp/src/bcrhp/pages/NewSite.vue

```diff
@@ -19,13 +19,15 @@ import SpatialLocation from './steps/Step3_SpatialLocation.vue';
 import SiteNames from './steps/Step4_SiteNames.vue';
 import RecognitionDetails from './steps/Step5_RecognitionDetails.vue';
 import SOS from './steps/Step6_SOS.vue';
-import UploadImage from './steps/Step7_UploadImage.vue';
+import UploadImageStep from './steps/Step7_UploadImage.vue';
 import SiteClassification from './steps/Step8_SiteClassification.vue';
 import SiteDetails from './steps/Step9_SiteDetails.vue';
 import SupportingDocuments from './steps/Step10_SupportingDocuments.vue';
 
 import { getHeritageSite } from '@/bcrhp/schema/HeritageSiteSchema.ts';
 import { HeritageSite } from '@/bcrhp/schema/HeritageSiteSchema.ts';
+import { getUploadImage } from '@/bcrhp/schema/UploadImageSchema.ts';
```

Comment:
Rather than using the UploadImage directly here, this should be added to the HeritageSite schema, and the HeritageSite schema provided to the Step7_UploadImage step, like the CivicAddress is handled in the Address step - please see the `Step2_SiteAddress.vue` step as a reference.